tests/test_util.go

```diff
@@ -558,6 +560,54 @@ func createEqualVersionsPredicate(version driver.Version) func(driver.VersionInf
 	}
 }
 
+// clusterSidecarsEqualSpec returns nil if sidecars from spec and cluster match
+func waitUntilClusterSidecarsEqualSpec(t *testing.T, spec api.DeploymentMode, depl api.ArangoDeployment) error {
+
+	c := cl.MustNewInCluster()
+	ns := getNamespace(t)
+
+	var noGood int
+	for start := time.Now(); time.Since(start) < 300*time.Second; {
```

Comment:
This feels very weird... 
Shouldn't we  use something similar to: https://github.com/arangodb/kube-arangodb/blob/cf338bd90649c65662ce6eedbc98fd84c50b8def/pkg/util/retry/retry.go#L124 (or L137 in the same File)?

Reply:
Possible, but is it worth doing?

Reply:
is offered under good practices